tomls/omnibus-base-sepolia-andromeda/spot/usdc.toml

```diff
@@ -1,5 +1,5 @@
 [setting.synthUsdcMaxMarketCollateral]
-defaultValue = "<%= parseEther(String(5_000_000)) %>"
+defaultValue = "<%= parseEther(String(10_000_000)) %>"
```

Comment:
need to adjust values in tests accordingly

Reply:
sure thing, i'll take a look

Reply:
@noisekit can you please trigger a re-run i dont think i have the necessary authorities

Reply:
There are some other values that need to be updated. I'll correct them soon

Reply:
fixed